var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -202,6 +204,25 @@ def configure_args(self):
 
         return args
 
+    def configure(self, spec, prefix):
+        if not self.spec.satisfies("platform=windows"):
+            super(Sqlite, self).configure(spec, prefix)
+
+    def build(self, spec, prefix):
+        if self.spec.satisfies("platform=windows"):
+            nmake = Executable("nmake.exe")
+            print(self.configure_flag_args)
+            nmake('CC = "%s"' % os.environ.get("SPACK_CC"), "Makefile.msc")
```

Comment:
Did you find this was necessary? We should already be setting CC in the environment: it generally points to the Spack compiler wrapper e.g. `<spack-prefix>/lib/spack/env/cc` and `SPACK_CC` points to the actual compiler path  - in that case we wouldn't get any of the extra `-L` flags inserted by Spack.